go/test/endtoend/vreplication/vstream_test.go

```diff
@@ -512,6 +514,196 @@ func testVStreamCopyMultiKeyspaceReshard(t *testing.T, baseTabletID int) numEven
 	return ne
 }
 
+// Validate that we can resume a VStream when the keyspace has been resharded
+// while not streaming. Ensure that there we successfully transition from the
+// old shards -- which are in the VGTID from the previous stream -- and that
+// we miss no row events during the process.
+func TestMultiVStreamsKeyspaceReshard(t *testing.T) {
```

Comment:
This should be added to config.json?

Reply:
It's run here -- all tests in that package are executed rather than a single one with `-run` elsewhere:
```
		"vtgate": {
			"File": "unused.go",
			"Args": ["vitess.io/vitess/go/test/endtoend/vtgate"],
			"Command": [],
			"Manual": false,
			"Shard": "vtgate_general_heavy",
			"RetryMax": 2,
			"Tags": []
		},
```

Reply:
Doh! Good catch! :-) I was thinking this was in the vtgate endtoend package where all of the tests in the package are executed rather than individual ones. I added it here: https://github.com/vitessio/vitess/pull/15395/commits/86749b775db2158e5733baddd9b4c3560d13470c

Thanks!